packages/client/hmi-client/src/components/widgets/tera-asset-enrichment.vue

```diff
@@ -114,17 +114,17 @@ function closeDialog() {
 
 const confirm = async () => {
 	isLoading.value = true;
+	closeDialog();
 
 	if (dialogType.value === DialogType.ENRICH) {
-		sendForEnrichment();
+		await sendForEnrichment();
```

Comment:
Why the need to add `await`?

Reply:
I want to call the `emit('finished-job')` (which refetches the dataset) after I get a response. I need to wait for the dataset to update so then I can fetch the updated version. Added a comment explaining this.